expect.go

```diff
@@ -392,7 +403,9 @@ func (e *Expect) Matcher(matcher func(*Response)) *Expect {
 // After creating request, all builders attached to Expect object are invoked.
 // See Builder.
 func (e *Expect) Request(method, path string, pathargs ...interface{}) *Request {
-	req := NewRequest(e.config, method, path, pathargs...)
+	req := NewRequest(e.config, e.context, method, path, pathargs...)
+	req.withContext(e.context)
```

Comment:
Could you clarify why do we need withContext() if we're already passing context to request constructor?

Reply:
that doesn't make sense indeed :) will fix :sweat_smile: